test_rpc.py

```
@@ -0,0 +1,47 @@
+import urllib.request, json
+
+def rpc(m, p):
+    r = urllib.request.Request('http://127.0.0.1:9850', method='POST',
+        headers={'Content-Type':'application/json'},
+        data=json.dumps({'jsonrpc':'2.0','method':m,'params':p,'id':1}).encode())
+    return json.loads(urllib.request.urlopen(r).read())
```

Ruff (0.15.7)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)